app/uk/gov/hmrc/bindingtariffclassification/model/Application.scala

```diff
@@ -61,14 +61,20 @@ case class LiabilityOrder
 case class EORIDetails
 (
   eori: String,
-  traderName: String,
+  traderName: String, // TODO: change to `name` because it is the name of the application holder (trader) or the name of the agent
```

Comment:
@ed0906  @pllaurado 
Since the EORIDetails is used by both the trader and holder, I believe we need to change the field `traderName` to `name` in here and also all front-ends

Reply:
We could argue they are both "trader"s though

Reply:
I think having it as `traderName` makes it clearer it is their business name (we could call it `businessName` or `companyName`) and not their personal name

Reply:
If the application comes from an agent, will `traderName` here represent the company name of the business holder or the name of the agent (or agent company)?

I am unsure about it

Reply:
We agreed with Aimee to call it `businessName` everywhere

Reply:
I think it is the company name of the agent, but worth checking with Matt